src/bio2bel/sources/intact.py

```diff
@@ -265,6 +272,19 @@ def _add_my_row(graph: BELGraph, row) -> None:
                     citation=pubmed_id,
                     evidence=EVIDENCE,
                 )
+            # oxidoreductase activity
+            elif relation == 'oxidoreductase activity electron transfer reaction':
+                graph.add_increases(
+                    source,
+                    target,
+                    citation=pubmed_id,
+                    evidence=EVIDENCE,
+                    subject_modifier=pybel.dsl.BiologicalProcess(
```

Comment:
haha does this not throw a value error? This is total nonsense

Reply:
hmm. should be better type annotations here

Reply:
Closed by 355c40c5c7b5cfe62c9bb4cb262d749596d87be1